foundry/pixi/containers/canvasLayers/placeablesLayers/tokenLayer.d.ts

```diff
@@ -1 +1,142 @@
-declare class TokenLayer extends PlaceablesLayer<Token> {}
+/**
+ * The Tokens Container
+ */
+declare class TokenLayer extends PlaceablesLayer<Token> {
+  constructor();
+
+  /**
+   * The current index position in the tab cycle
+   */
+  protected _tabIndex: number | null;
+
+  /**
+   * Remember the last drawn wildcard token image to avoid repetitions
+   */
+  protected _lastWildcard: string | null;
+
+  /**
+   * @override
+   * @defaultValue
+   * ```
+   * mergeObject(super.layerOptions, {
+   *   canDragCreate: false,
+   *   controllableObjects: true,
+   *   rotatableObjects: true,
+   *   gridPrecision: 1,
+   *   objectClass: Token,
+   *   sheetClass: TokenConfig,
+   *   zIndex: 100
+   * })
+   * ```
+   */
+  static get layerOptions(): PlaceablesLayer.LayerOptions;
+
+  /**
+   * Token objects on this layer utilize the TokenHUD
+   */
+  get hud(): TokenHUD;
+
+  /**
+   * An Array of tokens which belong to actors which are owned
+   */
+  get ownedTokens(): Token[];
+
+  /** @override */
+  tearDown(): Promise<void>;
+
+  /**
+   * @override
+   * @remarks Returns void
+   */
+  activate(): any;
+
+  /**
+   * @override
+   * @remarks Returns void
+   */
+  deactivate(): any;
+
+  /**
+   * Initialize the TokenLayer by preparing all Token sources which appear in the Scene.
+   */
+  initialize(): void;
+
+  /**
+   * @override
+   * @remarks Returns void
+   * */
+  controlAll(options?: PlaceableObject.ControlOptions): any;
```

Comment:
Should be fixed in parent.

Reply:
The parent is correct. This is an improper override, and the only overload. Typing this as all affects the fewest number of classes compared to typing the parent as unknown.